system/jhtree/jhtree.cpp

```diff
@@ -618,6 +614,12 @@ class CNodeMRUCache : public CMRUCacheOf<CKeyIdAndPos, CJHTreeNode, CNodeMapping
         // remove LRU until !full
         do
         {
+            //Never evict an entry that hasn't yet loaded - otherwise the sizeInMem can become inconsistent
+            CNodeMapping *tail = mruList.tail();
+            assertex(tail);
+            if (!tail->queryElement().isReady() )
+                break;
```

Comment:
not important afaict, but the implication is that that when add is called and full, there could be a few nodes that are in the process of loading at tail, than prevent add->makespace reducing amount used, so goes above mem. limit for a short time... until a future add that calls makespace and reduces below limit again. ?

If true, perhaps a comment here?